### PR TITLE
Translation to pt-br and loadmodel for low specs

### PR DESCRIPTION
**Pull request by mur4i**
## Description

Translation to pt-br and loadmodel for low specs

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by BerkieBb — CHANGES_REQUESTED** [review]
Change the IsPedInAnyVehicle checks to `cache.vehicle` and the dead checks to the dead check export, `exports.qbx_medical:isDead()`

File: client/main.lua [hunk 1]
```diff
@@ -507,7 +523,7 @@ end
 ---@param coords vector4
 ---@return number vehicleEntity
 local function createShowroomVehicle(model, coords)
-    lib.requestModel(model, 1000)
+    lib.requestModel(model, 5000)
```
Comment (artur-michalak) [inline client/main.lua:526]: Perhabs may be worth use requestModelTimeout

File: client/main.lua [hunk 2]
```diff
@@ -29,13 +29,13 @@ end
 
 local function confirmationCheck()
     local alert = lib.alertDialog({
-        header = 'Wait a minute!',
-        content = 'Are you sure you wish to proceed?',
+        header = 'Espere um minuto!',
```
Comment (ArturMichalak) [inline client/main.lua:32]: These are not language files. The default language for prompts is English.
If you want to translate this messages, create keys in locales/en.json and locales/es.json.

File: client/main.lua [hunk 3]
```diff
@@ -598,6 +614,8 @@ RegisterNetEvent('qbx_vehicleshop:client:testDrive', function(args)
     local plate = 'TEST'..lib.string.random('1111')
     local netId = lib.callback.await('qbx_vehicleshop:server:spawnVehicle', false, args.vehicle, testDrive.spawn, plate)
     testDriveVeh = netId
+    while not NetworkGetEntityFromNetworkId(netId) do Wait(100) end
```
Comment (artur-michalak) [inline client/main.lua:618]: lib.waitFor should be better option

File: locales/pt-br.json [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      "testdrive_timenoti": "Você tem %s minutos restantes",
+      "testdrive_complete": "Test drive do veículo concluído",
+      "paymentduein": "Seu pagamento do veículo vence em %s minutos",
+      "command_transfervehicle": "Presenteie ou venda seu veículo",
+      "command_transfervehicle_help": "ID do comprador",
+      "command_transfervehicle_amount": "Valor da venda (opcional)",
+      "transfervehicle_confirm": "Confirmar a troca de %s %s por $ %s"
+    }
+  }
+
```
Comment (artur-michalak) [inline locales/pt-br.json:84]:
```suggestion

```

Comment (artur-michalak) [reply locales/pt-br.json:84]: the last line should be empty

File: client/main.lua [hunk 5]
```diff
@@ -29,13 +29,13 @@ end
 
 local function confirmationCheck()
     local alert = lib.alertDialog({
-        header = 'Wait a minute!',
```
Comment (artur-michalak) [inline client/main.lua:32]: These are not language files. The default language for prompts is English.
If you want to translate this messages, create keys in locales/en.json and locales/es.json.

**Review by artur-michalak — CHANGES_REQUESTED** [review]

**Comment by Manason** [comment]
Closing as author has not been responsive. Feel free to re-open.

**Pull request closed without merging**